### ISSUE TITLE
More verbose output for errors

### ISSUE THREAD

**Issue by bykvaadm** [minio/minio#12482]
It's very unclear what's going on. I'm getting this kind of errors and I don't understand where to find the necessary file where those "evil" symbols are. grep'ed a lot, but unfortunate. I've never ever edited something by hands, only by mc client and this is unclear what symbol 'y' after top-level value and what's the } before "rsion" (Version?) and where to find wrong config.

```
Waiting for all MinIO sub-systems to be initialized.. lock acquired
Verifying if 11 buckets are consistent across drives...

API: SYSTEM()
Time: 08:58:01 UTC 06/10/2021
DeploymentID: f1cdbcc7-75aa-4c9f-8321-3401acb04a9f
Error: Unable to initialize config, some features may be missing invalid character 'y' after top-level value (*fmt.wrapError)
       3: cmd/server-main.go:390:cmd.initAllSubsystems()
       2: cmd/server-main.go:321:cmd.initServer()
       1: cmd/server-main.go:535:cmd.serverMain()
All MinIO sub-systems initialized successfully
Waiting for all MinIO IAM sub-system to be initialized.. lock acquired
Status:         16 Online, 0 Offline.

API: SYSTEM()
Time: 08:58:01 UTC 06/10/2021
DeploymentID: f1cdbcc7-75aa-4c9f-8321-3401acb04a9f
Error: Unable to migrate IAM users and policies to new format: cmd.iamFormat.readFieldHash: expect :, but found }, error found in #10 byte of ...|"}rsion"}}|..., bigger context ...|{"}rsion"}}|... (*fmt.wrapError)
       1: cmd/iam.go:647:cmd.(*IAMSys).Init()

API: SYSTEM()
Time: 08:58:01 UTC 06/10/2021
DeploymentID: f1cdbcc7-75aa-4c9f-8321-3401acb04a9f
Error: IAM sub-system is partially initialized, some users may not be available (*errors.errorString)
       1: cmd/iam.go:648:cmd.(*IAMSys).Init()
```

**Comment by harshavardhana** [comment]
This is because of a bug that we have reproduced and fixed @bykvaadm which version of MinIO are you running?

**Comment by bykvaadm** [comment]
minio/minio:RELEASE.2021-05-16T05-32-34Z

**Comment by harshavardhana** [comment]
> minio/minio:RELEASE.2021-05-16T05-32-34Z

you should upgrade to latest release this should be fixed already.  @bykvaadm

**Comment by bykvaadm** [comment]
minio/minio:RELEASE.2021-06-09T18-51-39Z

```
/usr/bin/docker-entrypoint.sh: line 97: docker_kms_master_encryption_env: command not found
Waiting for all MinIO sub-systems to be initialized.. lock acquired
Verifying if 11 buckets are consistent across drives...

API: SYSTEM()
Time: 09:27:31 UTC 06/10/2021
DeploymentID: f1cdbcc7-75aa-4c9f-8321-3401acb04a9f
Error: Unable to initialize config, some features may be missing invalid character 'y' after top-level value (*fmt.wrapError)
       3: cmd/server-main.go:392:cmd.initAllSubsystems()
       2: cmd/server-main.go:323:cmd.initServer()
       1: cmd/server-main.go:535:cmd.serverMain()
All MinIO sub-systems initialized successfully
Waiting for all MinIO IAM sub-system to be initialized.. lock acquired

API: SYSTEM()
Time: 09:27:31 UTC 06/10/2021
DeploymentID: f1cdbcc7-75aa-4c9f-8321-3401acb04a9f
Error: Unable to migrate IAM users and policies to new format: cmd.iamFormat.readFieldHash: expect :, but found }, error found in #10 byte of ...|"}rsion"}}|..., bigger context ...|{"}rsion"}}|... (*fmt.wrapError)
       1: cmd/iam.go:647:cmd.(*IAMSys).Init()

API: SYSTEM()
Time: 09:27:31 UTC 06/10/2021
DeploymentID: f1cdbcc7-75aa-4c9f-8321-3401acb04a9f
Error: IAM sub-system is partially initialized, some users may not be available (*errors.errorString)
       1: cmd/iam.go:648:cmd.(*IAMSys).Init()
```

**Comment by harshavardhana** [comment]
@bykvaadm did you have etcd, some kind of KMS etc configured?

**Comment by bykvaadm** [comment]
nope

**Comment by harshavardhana** [comment]
A quick fix here is `rm -rf <backend_drive>/.minio.sys/config` from all servers @bykvaadm and restart all servers. 

Can you tell us how you upgraded from which version to which version?

**Comment by bykvaadm** [comment]
minio/minio:RELEASE.2021-05-16T05-32-34Z -> RELEASE.2021-06-07T21-40-51Z

just by docker pull && restart

**Comment by harshavardhana** [comment]
> minio/minio:RELEASE.2021-05-16T05-32-34Z -> RELEASE.2021-06-07T21-40-51Z

I meant before upgrading to `minio/minio:RELEASE.2021-05-16T05-32-34Z`

**Comment by bykvaadm** [comment]
I have 16 disks in my cluster, should I rm folder on every server, every disk?
after rm this folder all users and policies would be destroyed? and I should create everything again?

**Comment by harshavardhana** [comment]
> I have 16 disks in my cluster, should I rm folder on every server, every disk?
> after rm this folder all users and policies would be destroyed? and I should create everything again?

@bykvaadm that is correct - also tell me what was the first release you were using and how have you upgraded?

**Comment by bykvaadm** [comment]
RELEASE.2021-03-17T02-33-02Z - initial release when cluster was created

**Comment by bykvaadm** [comment]
removed, restarted, the error is gone. but the latest tag has introduced new bug:
```
/usr/bin/docker-entrypoint.sh: line 97: docker_kms_master_encryption_env: command not found
```

it doesn't affect minio work, but..

**Comment by harshavardhana** [comment]
> ```
> /usr/bin/docker-entrypoint.sh: line 97: docker_kms_master_encryption_env: command not found
> ```

I just fixed it in c221633a8a7ecf59bec4b8bd0942ff0ac32e996f

**Comment by bykvaadm** [comment]
Thanks for help!
